anagrafica/templates/anagrafica_elenchi_delegati.html

```diff
@@ -0,0 +1,47 @@
+{% extends 'us_elenchi_inc_vuoto.html' %}
```

Comment:
Per la sede, è chiaro. sistemata in https://github.com/CroceRossaItaliana/jorvik/pull/405/commits/32335ec76279efde8689b281d78aa1b6597d6154